.github/workflows/osx_win_python_builds.yml

```diff
@@ -47,7 +47,7 @@ concurrency:
 
 env:
   # only run tmate if manually invoked and debug_enabled == true or a previous step failed
-  SETUP_TMATE: ${{ (((github.event_name == 'workflow_dispatch') && (github.event.inputs.debug_enabled == 'true')) || (github.job.status != 'success')) && (github.repository_owner == 'valhalla') }}
+  SETUP_TMATE: ${{ ((github.event_name == 'workflow_dispatch') && (github.event.inputs.debug_enabled == 'true')) && (github.repository_owner == 'valhalla') }}
```

Comment:
in the `env` section `github.job.status` doesn't really exist, so we push that into the `if`s further down